ast.go

```diff
@@ -145,7 +145,12 @@ func (fa *ASTFunc) generateParameters() []ast.Expr {
 		if p.Type.IsSlice && !p.Type.IsReturnArgument {
 			hasDeclaration = true
 
-			fa.addGoToCSliceConversion(p.Name, p.Type)
+			
+			if _, ok := p.Heuristics["pass_c"]; !ok {
```

Comment:
So I had a look at some functions of the Clang API and I think we can just use that as the default for arrays and slices. For example have a look at "clang_reparseTranslationUnit".